L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -6,15 +6,15 @@
 
 HGCalHistoSeedingImpl::HGCalHistoSeedingImpl(const edm::ParameterSet& conf)
```

Comment:
The logs and checks in the constructor should be revisited since `nBins1_` and `nBins2_` are not well defined at this stage